planning/behavior_path_planner/config/pull_over/pull_over.param.yaml

```diff
@@ -1,26 +1,54 @@
 /**:
   ros__parameters:
     pull_over:
+      request_length: 100.0
+      th_arrived_distance_m: 1.0
+      th_stopped_velocity_mps: 0.01
+      th_stopped_time_sec: 2.0  # It must be greater than the state_machine's.
+      pull_over_velocity: 2.0
+      pull_over_minimum_velocity: 0.3
+      margin_from_boundary: 0.5
+      decide_path_distance: 10.0
+      min_acc: -0.5
+      enable_shift_parking: true
+      enable_arc_forward_parking: true
+      enable_arc_backward_parking: false
+      # goal research
+      search_priority: "efficient_path" # "efficient_path" or "close_goal"
+      enable_goal_research: true
+      forward_goal_search_length: 20.0
+      backward_goal_search_length: 20.0
+      goal_search_interval: 1.0
+      goal_to_obj_margin: 2.0
+      # occupancy grid map
+      collision_check_margin: 1.0
+      theta_size: 360
+      obstacle_threshold: 60
+      # shift path
+      pull_over_sampling_num: 4
+      maximum_lateral_jerk: 2.0
+      minimum_lateral_jerk: 0.5
+      deceleration_interval: 15.0
+      maximum_deceleration: 1.0
+      after_pull_over_straight_distance: 5.0
+      before_pull_over_straight_distance: 5.0
+      # parallel parking path
+      after_forward_parking_straight_distance: 2.0
+      after_backward_parking_straight_distance: 2.0
+      forward_parking_velocity: 0.3
+      backward_parking_velocity: -0.3
+      arc_path_interval: 1.0
+      # hazard. Not used now.
+      double hazard_on_threshold_dis: 1.0
+      double hazard_on_threshold_vel: 0.5
```

Comment:
no need `double` ?

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/873/commits/652e424f8a879c8f26a6c4bcd4edd724313b5420